mobile/lib/main.dart

```diff
@@ -57,139 +59,110 @@ class MainPage extends StatelessWidget {
-            crossAxisAlignment: CrossAxisAlignment.stretch,
+      child: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.only(bottom: RaPageConstraints.radioPlayerHeight),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
             children: [
-              /// Ramówka widget
-              const Padding(
-                padding: _widgetPadding,
-                child: RamowkaWidget(),
-              ),
-
-              /// Old Ramowka
-              Padding(
-                padding: _widgetPadding,
-                child: ColorShadowedCard(
-                  shadowColor: context.colors.highlightYellow,
-                  header: Padding(
-                    padding: const EdgeInsets.symmetric(vertical: 4),
-                    child: Text(
-                      'Ramówka na dziś',
-                      style: context.textStyles.textMedium,
-                    ),
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch,
+                children: [
+                  /// Ramówka widget
+                  const Padding(
+                    padding: _widgetPadding,
+                    child: RamowkaWidget(),
                   ),
-                  footer: Padding(
-                    padding: const EdgeInsets.only(top: 4),
-                    child: Row(
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      children: [
-                        Padding(
-                          padding: const EdgeInsets.symmetric(
-                            horizontal: 2,
-                          ),
-                          child: Container(
-                            width: 8,
-                            height: 8,
-                            decoration: const BoxDecoration(
-                              color: Colors.grey,
-                              shape: BoxShape.circle,
-                            ),
-                          ),
+                  Padding(
+                    padding: _widgetPadding,
+                    child: ColorShadowedCard2(
+                      shadowColor: context.colors.highlightRed,
+                      footer: DefaultTextStyle(
+                        style: context.textStyles.textSmall.copyWith(
+                          color: context.colors.highlightGreen,
                         ),
-                        Padding(
-                          padding: const EdgeInsets.symmetric(
-                            horizontal: 2,
-                          ),
-                          child: Container(
-                            width: 8,
-                            height: 8,
-                            decoration: const BoxDecoration(
-                              color: Colors.white,
-                              shape: BoxShape.circle,
+                        child: Padding(
+                          padding: const EdgeInsets.all(10),
```

Comment:
Not a multiple of 2 - generally stick to 8 or 12 (or, in other cases, 4 and 16)
```suggestion
                          padding: const EdgeInsets.all(8),
```